components/builder-minio/habitat/default.toml

```diff
@@ -32,20 +17,29 @@ expected = 2
 #   distributed setup are encoded as HTTP(s) URIs.
 #
 # ~members~ is a ~Array~ of DIRs
-# Ignored when standalone
+# When members is empty standalone mode will be used with svc_data_path
 members = []
 
-# Admin Account
-key_id = "depot"
-secret_key = "password"
+# Default bucket name to create
+bucket_name = "habitat-builder-artifact-store.default"
 
-# Enable SSL in your Minio
-# This expects certificates files loaded via ~hab file upload~ or
-# in $pkg_svc_config_path/certs: ~private.key~ and ~public.crt~
-use_ssl = false
+# Bind to a specific ADDRESS (can be an IP or hostname). Default - all
+bind_address = ""
+# Bind to a specific PORT
+bind_port = 9000
 
-# Password for certificate
-ssl_cert_pw = ""
+# Setup additional environment variables to pass to minio
+# Doc: https://docs.min.io/docs/minio-server-configuration-guide.html
+[env]
+# Set minio admin user and password
+MINIO_ACCESS_KEY = "habitat-depot"
+MINIO_SECRET_KEY = "habitat-password"
```

Comment:
This needs to be changed back to "password" to avoid breaking our default envvars for on-prem.